probe/main.go

```diff
@@ -153,3 +176,16 @@ func interrupt() chan os.Signal {
 	signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
 	return c
 }
+
+func interfaces() string {
```

Comment:
I'm not sure capturing on all interfaces is a good idea.  Will this capture on the docker bridge and eth0?  Will packets get captured twice?